concepts.tex

```diff
@@ -404,17 +409,17 @@
 // Requires: \tcode{std::forward<T>(t)} shall be swappable with \tcode{std::forward<U>(u)}.
 template <class T, class U>
 void value_swap(T&& t, U&& u) {
-  using std::swap;
-  swap(std::forward<T>(t), std::forward<U>(u)); // OK: uses ``swappable with'' conditions
-                                                // for rvalues and lvalues
+  @\oldtxt{using std::swap;}@
+  @\newtxt{ranges::}@swap(std::forward<T>(t), std::forward<U>(u)); // OK: uses ``swappable with'' conditions
+                                                        // for rvalues and lvalues
 }
 
 // Requires: lvalues of \tcode{T} shall be swappable.
 template <class T>
 void lv_swap(T& t1, T& t2) {
-  using std::swap;
-  swap(t1, t2);                                 // OK: uses swappable conditions for
-}                                               // lvalues of type \tcode{T}
+  @\oldtxt{using std::swap;}@
```

Comment:
I use `oldoldtxt` when changing official working draft wording so I know what parts should end up in red strikethrough. It makes my job as editor much easier.